netket/experimental/driver/tdvp.py

```diff
@@ -532,6 +539,48 @@ def odefun(state: MCState, driver: TDVP, t, w, *, stage=0):  # noqa: F811
     return driver._dw
 
 
+@partial(jax.jit, static_argnums=(3, 4))
```

Comment:
what does this function do?

Reply:
Essentially computing the RHS of the ODE from the forces. Its `Re[-i F]` for the real-time real-parameter case, `-i F` for real-time complex parameters, etc.

Maybe we can generalize it in the future to also support `Im[-i F]` (TDVP instead of McLachlan's VP) and similar stuff.